src/civic/index.js

```diff
@@ -20,9 +20,6 @@ const { downloadVariantRecords, processVariantRecord } = require('./variant');
 
 const ajv = new Ajv();
 
-
-const TRUSTED_CURATOR_ID = 968;
```

Comment:
was there already one civic curator with trusted status, and this change just allows for any number of trusted curators?

Reply:
yup, but we have now generalised this so external people can pick/choose their trusted curators